src/components/Features.jsx

```diff
@@ -32,118 +32,122 @@ const Container = styled.div`
 `;
 
 const DecorationContainer = styled.div`
-display: flex;
-flex-wrap: wrap;
-gap: 2rem;
-justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2rem;
+  justify-content: center;
 `;
 
 const DecorationData = styled.div`
-text-align: center;
-padding: 1rem 1rem 2rem;
-background-color: var(--container-color);
-box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
-border-radius: 1rem;
-width: calc(25% - 1rem);
-max-width: 220px;
-flex: 1 0 auto;
-box-sizing: border-box;
+  text-align: center;
+  padding: 2rem;
+  background: rgba(255, 255, 255, 0.1);
+  box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
+  backdrop-filter: blur(10px);
+  border-radius: 1rem;
+  width: calc(25% - 1rem);
+  max-width: 250px;
+  flex: 1 0 auto;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
 `;
 
 const DecorationImg = styled.img`
-  height : 250px;
+  height: 250px;
+  width: 100%;
+  object-fit: cover;
+  border-radius: 0.5rem;
 `;
 
 const DecorationTitle = styled.h3`
-  font-size: 20px;
-
-  margin-bottom: 1rem;
-  margin-right : 0.5rem;
-  margin-left : 0.5rem;
-  margin-top : 1rem;
-  
+  font-size: 1.5rem;
+  margin: 1rem 0;
+  color: var(--title-color);
 `;
 
 const Button = styled.div`
   display: inline-block;
-  background-color: rgb(48, 175, 91);
+  background-color: rgba(48, 175, 91, 0.85);
   color: white;
   padding: 1rem 1.5rem;
   border-radius: 0.5rem;
   font-weight: var(--font-semi-bold);
   transition: 0.3s;
   cursor: pointer;
   text-decoration: none;
+  margin-top: 1rem;
 
   &:hover {
-    background-color: rgb(43, 57, 74);
+    background-color: rgba(43, 57, 74, 0.85);
```

Comment:
why is this change necessary?

Reply:
It's to make a color a lil bit transparent